packages/salesforcedx-vscode-lightning/package.json

```diff
@@ -24,13 +24,18 @@
     "Programming Languages"
   ],
   "dependencies": {
-    "@salesforce/salesforcedx-slds-linter": "45.7.0",
```

Comment:
Does this mean that the existing slds functionality is being replaced by either `aura-language-server` or `lightning-lsp-common` ?

Since the use of `salesforcedx-slds-linter` is going away, can we also delete that module ?

Reply:
lightning-lsp-common is just a package with common code for both aura and lwc lsps. We are removing the slds-linter as per @ntotten 's request. There is a new version in the works that I believe will be put in soon. But for now the existing one doesn't work and hasn't been maintained. The new slds linter probably belongs in its own package I would think.

I can remove the slds-linter package as well, just wasn't sure if you'd be ok with that or if I should leave it until its fully replaced

Reply:
To be clear though, with my existing changes, the slds-linter will no longer be present in VSCode

Reply:
> I can remove the slds-linter package as well, just wasn't sure if you'd be ok with that or if I should leave it until its fully replaced

Let's remove it. The replacement is a complete rewrite anyway.